src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -306,12 +318,14 @@ private void AddNameAndTypeArgumentsOrParameters(INamedTypeSymbol symbol)
             {
                 // If top level tuple uses non-default names, there is no way to preserve them
                 // unless we use tuple syntax for the type. So, we give them priority.
-                if (HasNonDefaultTupleElements(symbol) || CanUseTupleTypeName(symbol))
+                if (!format.CompilerInternalOptions.IncludesOption(SymbolDisplayCompilerInternalOptions.UseValueTuple))
```

Comment:
Should VB `SymbolDisplay` support `UseValueTuple` as well? Perhaps add `// PROTOTYPE` comment for now.

Reply:
Good point. Filed https://github.com/dotnet/roslyn/issues/26401